spec/requests/api/v1/topics/index_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe 'GET api/v1/topics', type: :request do
+  let(:password)  { 'password' }
+  let(:user)      { create(:user, password: password) }
+  let!(:topic)    { create(:topic) }
```

Comment:
What do you about the approach mentioned by @gabrielbursztein2 about creating multiple topics in this test? I think that you should do it, then you should expect to get them all since this is the `index` endpoint.

Use [create_list](https://www.rubydoc.info/gems/factory_bot/file/GETTING_STARTED.md#Building_or_Creating_Multiple_Records) to create multiple object for a given factory